source/codegen/metadata/nidaqmx/config.py

```diff
@@ -15,6 +15,7 @@
         'const char[]': 'string',
         'float64': 'double',
         'float64[]': 'repeated double',
+        'const float64[]': 'repeated double',
```

Comment:
Note: @gregstoll is working on a change that will remove the need for this duplication.